s2fft/recursions/turok.py

```diff
@@ -227,41 +235,43 @@ def compute_quarter_slice(
     return dl
 
 
-def compute_quarter(dl: np.ndarray, beta: float, l: int, L: int) -> np.ndarray:
-    """Compute the left quarter triangle of the Wigner-d matrix via Turok & Bucher
+def compute_quarter(dl: np.ndarray, beta: float, ell: int, L: int) -> np.ndarray:  # noqa: C901
+    """
+    Compute the left quarter triangle of the Wigner-d matrix via Turok & Bucher
     recursion.
 
     Args:
         dl (np.ndarray): Wigner-d matrix slice to populate (shape: 2L-1, 2L-1).
 
         beta (float): Polar angle in radians.
 
-        l (int): Harmonic degree of Wigner-d matrix.
+        ell (int): Harmonic degree of Wigner-d matrix.
```

Comment:
Ruff flagged this variable name as ambiguous (https://docs.astral.sh/ruff/rules/ambiguous-variable-name/) so I manually changed from `l` to `ell` here and in another function below.